python/splitter.py

```diff
@@ -66,8 +83,8 @@ def __main__():
     print("{}%".format(extra_fee_percentage*100) )
 
     splits = []
-
-    for i in range(int(participants)):
+    
+    for i in range(1, int(participants)+1):
```

Comment:
You shouldn't change the index, incase you want to use this later and forget you may get out of bounds errors. Instead change the print statement to handle the change

Reply:
it's scoped just for that for loop and not used for anything else, but okay